pkgs/applications/system/asusctl/default.nix

```diff
@@ -13,13 +13,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "asusctl";
-  version = "5.0.7";
+  version = "5.0.10";
 
   src = fetchFromGitLab {
     owner = "asus-linux";
     repo = "asusctl";
     rev = version;
-    hash = "sha256-thTzNB6GmzHG0BaaacmmQogRrLK1udkTYifEivwDtjM=";
+    hash = "sha256-H8x3nfOFRv9DkbDkFw+LO1tdHiVyU3SzetqED4twPSk=";
   };
 
   cargoHash = "";
```

Comment:
```suggestion
```

Don't know why it's there

Reply:
dont know either, it was there before

Reply:
Looks like it was added by mistake. Please remove it.

Reply:
Done